locale/English.json

```diff
@@ -46,8 +46,9 @@
       "btnRisky": "(Dangerous!)",
       "qrContentCopied": "Copied QR Content",
       "translatedText": "Translated Text",
-      "feedbackSuccess":"You have Successfully Submitted Feedback.Thank You",
-      "feedbackFailure": "Something went wrong...Please try again "
+      "feedbackSuccess": "You have Successfully Submitted Feedback.Thank You",
```

Comment:
Space after fullstop. and no caps inbetween.

"You have successfully submitted feedback. Thank You"